Sources/MapboxMaps/Foundation/LogoView.swift

```diff
@@ -27,13 +27,20 @@ public class LogoView: UIView {
         }
     }
 
-    internal var logoSize: LogoSize
+    internal var logoSize: LogoSize {
+        didSet {
+            invalidateIntrinsicContentSize()
+        }
+    }
+
+    public override var intrinsicContentSize: CGSize {
```

Comment:
For a follow-up PR: let's make everything that's currently public in LogoView internal